docker/test/integration/minifi/core/MqttBrokerContainer.py

```diff
@@ -0,0 +1,23 @@
+import logging
+from .Container import Container
+
+
+class MqttBrokerContainer(Container):
+    def __init__(self, name, vols, network, image_store):
+        super().__init__(name, 'mqtt-broker', vols, network, image_store)
+
+    def get_startup_finished_log_entry(self):
+        return "mosquitto version 2.0.11 running"
```

Comment:
Could this be changed somehow not to be version dependent?

Reply:
With regex matching I think. Would you second that?
Currently the output of starting mosquitto in verbose:
```
adam@amarkovics-Linux:~/work/minifi-cpp/build$ mosquitto -v
1632119285: mosquitto version 2.0.12 starting
1632119285: Using default config.
1632119285: Starting in local only mode. Connections will only be possible from clients running on this machine.
1632119285: Create a configuration file which defines a listener to allow remote access.
1632119285: For more details see https://mosquitto.org/documentation/authentication-methods/
1632119285: Opening ipv4 listen socket on port 1883.
1632119285: Opening ipv6 listen socket on port 1883.
1632119285: mosquitto version 2.0.12 running
```

Reply:
If that's not a huge effort to implement regex check here, that would okay.